apps/meteor/client/components/Omnichannel/modals/EnterpriseDepartmentsModal.tsx

```diff
@@ -23,8 +23,7 @@ const EnterpriseDepartmentsModal = ({ closeModal }: { closeModal: () => void }):
 		closeModal();
 	};
 
-	const tabTypeIsUpgradeYourPlan =
-		tabType === 'go-fully-featured' || tabType === 'go-fully-featured-registered' || tabType === 'upgrade-your-plan';
+	const isTypeUpgradeYourPlan = tabType === 'go-fully-featured' || tabType === 'go-fully-featured-registered';
```

Comment:
Can we use startsWith or an array of options? In case we ever need to add more checks

Like [gofullyfeatured].includes(tabType)